commercetools/commercetools-adyen-integration
# Add notification api

### How to run:
##### Set the following env variables:

Var Name | value | default value
------------ | ------------- | -------------
CTP_PROJECT_KEY |  | 
CTP_CLIENT_ID | | 
CTP_CLIENT_SECRET | | 
LOG_LEVEL | | debug
PORT | | 443

##### Run : 
```
npm run start
```

If you want to test with Adyen locally:
- run `ngrok http 443`
- go to Adyen Customer Area (https://ca-test.adyen.com) > Account > Server communication
- Click "Add" in front of the "Standard Notification" line
- Paste the url generated by ngrok in the URL field in the opened form
- Select notifications you want to test
- Click on "Test Configuration" button

Adyen should send multiple notifications to the provided URL.

### TODO:

- [x] Core logic implementation
